systems/trajectory_optimization/hybrid_dircon.cc

```diff
@@ -35,24 +35,21 @@ HybridDircon<T>::HybridDircon(const MultibodyPlant<T>& plant,
                               vector<double> maximum_timestep,
                               vector<DirconKinematicDataSet<T>*> constraints,
                               vector<DirconOptions> options)
-    : MultipleShooting(plant.num_actuators(),
-                       plant.num_positions() + plant.num_velocities(),
-                       std::accumulate(num_time_samples.begin(),
-                                       num_time_samples.end(),
-                                       0) - num_time_samples.size() + 1,
-                       1e-8,
-                       1e8),
+    : MultipleShooting(
+          plant.num_actuators(), plant.num_positions() + plant.num_velocities(),
+          std::accumulate(num_time_samples.begin(), num_time_samples.end(), 0) -
+              num_time_samples.size() + 1,
+          1e-8, 1e8),
       plant_(plant),
       constraints_(constraints),
       num_modes_(num_time_samples.size()),
       mode_lengths_(num_time_samples),
       v_post_impact_vars_(NewContinuousVariables(
-          plant.num_velocities() *
-              (num_time_samples.size() - 1),
-          "v_p")) {
+          plant.num_velocities() * (num_time_samples.size() - 1), "v_p")) {
   DRAKE_ASSERT(minimum_timestep.size() == num_modes_);
   DRAKE_ASSERT(maximum_timestep.size() == num_modes_);
   DRAKE_ASSERT(constraints.size() == num_modes_);
+  DRAKE_ASSERT(options.size() == num_modes_);
```

Comment:
This line and the next three places which use `SetConstraintScaling` are the changes to this file beside code style reformatting.